packages/autorest.typescript/test/rlcIntegration/lroRest.spec.ts

```diff
@@ -273,13 +286,13 @@ describe("LRO Rest Client", () => {
       const poller = await getLongRunningPoller(client, initialResponse, {
         intervalInMs: 0
       });
-
       const result = await poller.pollUntilDone();
       if (isUnexpected(result)) {
         const error = `Unexpected status code ${result.status}`;
         assert.fail(error);
-        throw new Error(error);
       }
+      // FIXME
```

Comment:
@deyaaeldeen We have 8 cases failure after enabling this flag. Could you help on that?



https://dev.azure.com/azure-sdk/public/_build/results?buildId=2156796&view=logs&j=71cc2e7d-6454-5bf0-71b5-f1eef07a6c23&t=f2857310-5d68-5640-1409-0ba88dcfe076&l=1727

Reply:
I am looking now but I suspect the test expectation should be updated.
For the first failure, the operation succeeded, see [the corresponding core-lro test](https://github.com/Azure/azure-sdk-for-js/blob/6a76e6ac3f67fdf7b2d22c7b3a242538b0c60f54/sdk/core/core-lro/test/lro.spec.ts#L302).

Reply:
I updated these test cases and then the relevant behaviors are not as my expectation.

Reply:
I updated expectations in https://github.com/Azure/autorest.typescript/commit/c989dd5e7044e2b9c2c2173751faf53a22cb52f8